### ISSUE TITLE
Quality of projects / Workspace enviroment

### ISSUE THREAD

**Issue by StjepanBM1** [ringwormGO-organization/ringOS#28]
**TL;DR A lot of ringwormGO projects are rushed because of unrealistic deadlines (cutting corners included).**

As a Project Director of ringwormGO I noticed there is a lot of toxicity around here. For example -> a lot of projects are rushed because of this they don't have that bit of quality that is expected. Most people working on ringwormGO usually are worikng on multiple projects at once, as a result there is a lot of pressure to hit deadlines (even though they are unrealistic), meaning that projects often need a lot of updates after the release (don't get me started about us cutting corners) and aren't good as they were imagined / planned to be. Usually most of these projects have stolen code (because of deadlines). I do think that we should slow down when it comes to updating / creating projects. If we do slow down expect updates on projects to happen every 3 days or so.

What do you think?

**Comment by Andrej123456789** [comment]
Please, we have issue at main repo

**Comment by Andrej123456789** [comment]
Termi will be on pause until ringOS version 2.

For all other send comment on issue on main repo
